packages/core/helper-plugin/src/utils/isAbsoluteUrl/tests/index.test.js

```diff
@@ -0,0 +1,19 @@
+import isAbsoluteUrl from '../index';
+
+describe('isAbsoluteUrl', () => {
+  test('can check is a provided url is absolute or not', () => {
```

Comment:
Could you please use `test.each` instead (or a loop) to keep this a bit more concise? https://jestjs.io/docs/api#testeachtablename-fn-timeout

Reply:
yes sure

Reply:
I removed the test and the utility